addOns/network/src/test/java/org/zaproxy/addon/network/GlobalExclusionsOptionsUnitTest.java

```diff
@@ -247,4 +307,74 @@ private static ZapXmlConfiguration configWith(String value) {
         }
         return config;
     }
+
+    private static String getNoVersionConfig() {
+        return """
+               <globalexcludeurl>
+                   <url_list>
+                       <url>
+                             <regex>^.*\\.(?:gif|jpe?g|png|ico|icns|bmp)$</regex>
+                             <description>Extension - Image (ends with .extension)</description>
+                             <enabled>true</enabled>
+                         </url>
+                         <url>
+                             <regex>^[^\\?]*\\.(?:gif|jpe?g|png|ico|icns|bmp)\\?.*$</regex>
+                             <description>ExtParam - Image (extension plus ?params=values)</description>
+                             <enabled>false</enabled>
+                         </url>
+                         <url>
+                             <regex>^[^\\?]*\\.(?:mp[34]|mpe?g|m4[ap]|aac|avi|mov|wmv|og[gav])\\?.*$</regex>
+                             <description>ExtParam - Audio/Video (extension plus ?params=values)</description>
+                             <enabled>false</enabled>
+                         </url>
+                         <url>
+                             <regex>^.*foo$</regex>
+                             <description>Not Default</description>
+                             <enabled>false</enabled>
+                         </url>
+                         <url>
+                             <regex>^[^\\?]*/(?:WebResource|ScriptResource)\\.axd\\?d=.*$</regex>
+                             <description>ExtParam - .NET adx resources (SR/WR.adx?d=)</description>
+                             <enabled>false</enabled>
+                         </url>
+                       </url_list>
+                   <confirmRemoveToken>true</confirmRemoveToken>
+               </globalexcludeurl>""";
+    }
+
+    private static String getVersion1Config() {
+        return """
+               <network>
+                   <globalExclusions version=\"1\">
+                       <exclusions>
+                           <confirmRemove>true</confirmRemove>
+                           <exclusion>
+                               <name>Extension - Image (ends with .extension)</name>
+                               <value>^.*\\.(?:gif|jpe?g|png|ico|icns|bmp)$</value>
+                               <enabled>true</enabled>
+                           </exclusion>
+                           <exclusion>
+                               <name>ExtParam - Image (extension plus ?params=values)</name>
+                               <value>^[^\\?]*\\.(?:gif|jpe?g|png|ico|icns|bmp)\\?.*$</value>
+                               <enabled>false</enabled>
+                           </exclusion>
```

Comment:
One image is enough.

Reply:
No because we ensure it's a matched pattern/value first.

Reply:
Sorry, what's this testing then?

Reply:
This may be related to my failure to understand the other/earlier comment. I'll have to double check

Reply:
Simply that the changes are applied as expected.

I guess your point is that we only need to test that once... 😀

Reply:
Right :)

Reply:
Done